### ISSUE TITLE
Parity 1.11.10 stuck syncing ancient block after warp sync

### ISSUE THREAD

**Issue by boris-savic** [paritytech/parity-ethereum#9486]
- **Parity Ethereum version**: 1.11.10-stable
- **Operating system**:  Ubuntu 18.04
- **Installation**: one-line installer
- **Fully synchronized**:  no
- **Network**: ethereum 
- **Restarted**: yes

I've setup a new node with more or less default config (attached the actual config.toml at the end) and after warp sync the node is stuck constantly fetching block 57. I've restarted the node several times, reinstalled Parity, etc. 

Logs:
```
Sep 06 12:52:51 parity-node parity[17017]: 2018-09-06 12:52:51  Imported #6281861 0x9006…b89e (71 txs, 8.00 Mgas, 179 ms, 13.81 KiB)
Sep 06 12:52:53 parity-node parity[17017]: 2018-09-06 12:52:53  Imported #6281862 0x2b8f…0b16 (62 txs, 8.00 Mgas, 162 ms, 13.60 KiB)
Sep 06 12:53:02 parity-node parity[17017]: 2018-09-06 12:53:02         #57    8/50 peers    185 MiB chain  234 MiB db  0 bytes queue  353 MiB sync  RPC:  0 conn,  0 req/s, 416 µs
Sep 06 12:53:23 parity-node parity[17017]: 2018-09-06 12:53:23  Imported #6281863 0x39e5…2d4c (29 txs, 7.98 Mgas, 116 ms, 4.79 KiB)
Sep 06 12:53:28 parity-node parity[17017]: 2018-09-06 12:53:28  Imported #6281864 0x5b5d…5dcd (88 txs, 7.99 Mgas, 228 ms, 13.38 KiB)
Sep 06 12:53:34 parity-node parity[17017]: 2018-09-06 12:53:34  Imported #6281865 0xb8ce…3cd2 (74 txs, 7.99 Mgas, 237 ms, 29.24 KiB)
Sep 06 12:53:37 parity-node parity[17017]: 2018-09-06 12:53:37         #57    8/50 peers    185 MiB chain  234 MiB db  0 bytes queue  354 MiB sync  RPC:  0 conn,  0 req/s, 416 µs
Sep 06 12:53:39 parity-node parity[17017]: 2018-09-06 12:53:39  Imported #6281866 0xc50f…6683 (60 txs, 7.99 Mgas, 174 ms, 9.25 KiB)
Sep 06 12:53:50 parity-node parity[17017]: 2018-09-06 12:53:50  Imported #6281867 0x1eb1…40e3 (190 txs, 7.98 Mgas, 312 ms, 39.11 KiB)
Sep 06 12:53:59 parity-node parity[17017]: 2018-09-06 12:53:59  Imported #6281868 0x1115…7231 (67 txs, 7.99 Mgas, 164 ms, 11.96 KiB)
Sep 06 12:54:01 parity-node parity[17017]: 2018-09-06 12:54:01  Imported #6281869 0x6dcd…0352 (42 txs, 7.97 Mgas, 141 ms, 10.02 KiB)
Sep 06 12:54:12 parity-node parity[17017]: 2018-09-06 12:54:12         #57    8/50 peers    185 MiB chain  233 MiB db  0 bytes queue  358 MiB sync  RPC:  0 conn,  0 req/s, 416 µs
```

As you can see from the logs it's constantly syncing block 57. I've tried deleting the database completely and resyncing and it still gets stuck at the same block. 

My **config.toml**: 

```
[parity]
mode = "active"

[network]
min_peers = 50
max_peers = 250
allow_ips = "public"

[rpc]
server_threads = 2

[ipc]
disable = true

[footprint]
cache_size = 4096
```

**Comment by boris-savic** [comment]
After investigating further, I believe this is related to #9306 as I'm also getting invalid block hashes. 

Is there any simple solution to this or restarting until hopefully it gets connected to a good peer?

**Comment by 5chdn** [comment]
Thanks. This is an important addition to #9306 

I'll close this as duplicate though, please subscribe to the other issue.

**Comment by 5chdn** [comment]
we are currently isolating and purging the node, please try again later.

there is a separate fix for #9306 being worked on.
